localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -32,6 +32,38 @@ services:
       interval: 30s
       retries: 1
       timeout: 3s
+  cloud-nine-wallet-point-of-sale: 
+    hostname: cloud-nine-wallet-point-of-sale
+    image: rafiki-point-of-sale 
+    build: 
+      context: ../.. 
+      dockerfile: ./packages/point-of-sale/Dockerfile.dev 
+    restart: always 
+    networks: 
+      - rafiki 
+    ports: 
+      - '3007:3007' 
+    volumes: 
+      - type: bind 
+        source: ../../packages/point-of-sale/src 
+        target: /home/rafiki/packages/point-of-sale/src 
+        read_only: true 
+    environment: 
+      NODE_ENV: ${NODE_ENV:-development} 
+      INSTANCE_NAME: CLOUD-NINE 
+      TRUST_PROXY: ${TRUST_PROXY} 
+      LOG_LEVEL: debug 
+      CARD_SERVICE_PORT: 3007 
```

Comment:
Ditto with the happy-life-bank comment, need to update the env var name and port number.